src/handler.ts

```
@@ -43,13 +68,222 @@ function setupAWS() {
+      length: learningObject.length,
+      levels: learningObject.levels,
+      name: learningObject.name,
+      outcomes: learningObject.outcomes,
+      version: learningObject.version,
+      status: learningObject.status,
+    };
+    return learningObjectSearchDocument;
+}
+
+/**
+ * Updates the learning object ReadMes to reflect the new author
+ * change
+ * @param toObjects The objects updated
+ */
+async function updateLearningObjectReadMe(fromObjects: any) {
+    fromObjects.map(async learningObject => {
+        const learningObjectID = learningObject._id;
+        const options = {
+            uri: `${process.env.LEARNING_OBJECT_API}/learning-objects/${learningObjectID}/pdf`,
+            headers: {
+                Authorization: `Bearer ${generateServiceToken()}`,
+            },
+            method: 'PATCH',
+        };
+        request(options);
+    });
+}
+
+/**
+ * Change authorship of objects if it has children
+ * @param fromObjects objects that were changed
+ * @param newAuthorID id the new author
+ */
+async function moveLearningObjectChildren (fromObjects: any, newAuthorID: string, oldAuthorAccessID: string, newAuthorAccessID: string) {
+    const db = await MongoDB.getInstance();
+    try {
+        fromObjects.map(async learningObject => {
+            if (learningObject.children) {
+                let children = [];
+                for (let i = 0; i < learningObject.children.length; i++) {
+                    const objects = await db.updateLearningObjectAuthor(learningObject.children[i], newAuthorID);
+                    children.push(objects.value);
+                    children.map(async child => {
+                        const cuid = child.cuid;
+                        await copyFiles(cuid, oldAuthorAccessID, newAuthorAccessID);
+                    });
+                }
+            } else {
+                learningObject.children = [];
+            }
+        });
+    } catch (e) {
+        console.log(e);
```

Should also return this error.